omnibus/files/private-chef-cookbooks/private-chef/templates/default/openssl.cnf.erb

```diff
@@ -0,0 +1,361 @@
+#
+# OpenSSL example configuration file.
```

Comment:
I dont see a way to do any sort of file nesting, however we can cull down this config file to just what we need. Most of these are either already defaults by the library or not useful to us.